_data/challenges.yml

```diff
@@ -148,9 +149,9 @@
   sub: yes
   submit: https://www.codechef.com/SEID2017/problems/SEIDIA02
   prizes:
-    - name: Powerbank 
+    - name: Powerbank
       num: 1
-      link: 
+      link:
```

Comment:
@xicon73 This power bank does not have a link but the other one (right below) has. Aren't they the same? :)